projects/Eventful-API-test-page/index.js

```diff
@@ -1,17 +1,20 @@
-const connection = require('./connection');
+const pool = require('./connection');
 const app = require('./app');
 
- const pgConnect = () => {
-  connection.connect((err) =>{
+const pgConnect = () => {
+  pool.connect((err, client) => {
     if (err) throw err;
 
-     console.log('Welcome to Eventonica')
+    console.log('Welcome to Eventonica')
     console.log("connected as Administrator");
 
-     app.startQuestion(()=>{ connection.end() });
+    app.startQuestion(() => {
+      client.release();
```

Comment:
I don't think using callback for `pool.connect` requires you to manually release the client. It's also not clear to me why we're checking out a client here from the pool, since we don't do anything with it.

Reply:
Oh, is this supposed to be a template of sorts for the students to follow?

Reply:
This is starter code for the project, so yeah, the intention is for it to be a template. Ideally, students would check out a client and release it each time they query the db in app.js, but to get the project running they should successfully connect to the database to know it's working.  But you are correct that using a callback automatically releases the client.

Reply:
Alternatively, `connection.js` could create a `Client` instead of a `Pool`, and students could just use the one client in all of their queries. Although the node-postgres docs recommend using pools instead of individual clients

Reply:
No, I think using a pool is good practice.